getting-started/git-boot-camp.rst

```diff
@@ -44,11 +44,13 @@ You will only need to do this once.
 
 1. Go to https://github.com/python/cpython.
 
-2. Press ``Fork`` on the top right.
+2. Press ``Fork`` located near the top right of the page.
 
-3. When asked where to fork the repository, choose to fork it to your username.
+   * Uncheck the "Copy the ``main`` branch only" button.
```

Comment:
This is an action to be followed, let's promote it to a numbered item, and remember to renumber the following ones:
```suggestion
4. Uncheck the "Copy the ``main`` branch only" button.
```

Reply:
@hugovk I've updated as you've suggested.